internal/resources/kubernetessecret/scope/cluster_schema.go

```diff
@@ -0,0 +1,85 @@
+/*
+Copyright © 2023 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+*/
+
+package scope
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/vmware/terraform-provider-tanzu-mission-control/internal/helper"
+	secretclustermodel "github.com/vmware/terraform-provider-tanzu-mission-control/internal/models/kubernetessecret/cluster"
+)
+
+var ClusterPolicyFullname = &schema.Schema{
```

Comment:
please check the the variables names when you copy the code from other source files

Reply:
updated PTAL